cmd/all-in-one/all_in_one_test.go

```diff
@@ -70,10 +72,13 @@ func TestAllInOne(t *testing.T) {
 func createTrace(t *testing.T) {
 	req, err := http.NewRequest(http.MethodGet, getServicesURL, nil)
 	require.NoError(t, err)
-	req.Header.Add("jaeger-debug-id", "debug")
 
 	resp, err := httpClient.Do(req)
 	require.NoError(t, err)
+	traceResponse := resp.Header.Get("traceresponse")
+	traceID := strings.Split(traceResponse, "-")[1]
```

Comment:
nit: I'd do this in two steps so we can have an explicit assertion on the expected number of parts in the traceresponse:

```go
parts := strings.Split(traceResponse, "-")
require.Len(t, parts, 4) // [version] [trace-id] [child-id] [trace-flags]
traceID := parts[1]
```

Reply:
Nice observation. Thanks @albertteoh